src/firefox/remote.js

```diff
@@ -138,7 +141,7 @@ export class RemoteFirefox {
         }
         log.debug(
           `Remote Firefox has these addons: ${addons.map((a) => a.id)}`);
-        throw new WebExtError(
+        throw new UsageError(
```

Comment:
This should be a `WebExtError` because the installation process was done automatically by `web-ext`, not the developer. If the extension was not installed, it means there was a bug in `web-ext`. It is unlikely that this error will ever happen.